Documentation/PackageDescriptionV4.md

```diff
@@ -313,7 +313,7 @@ backwards-compatible manner.
 bug fixes.  
 \- *prereleaseIdentifiers*: Used to denote a pre-released version for eg:
```

Comment:
Drive-by comment: While you're here, could we fix up the grammar here and below a little bit? The changes would be--

In this line:

* Change `pre-released` to `pre-release`
* Change `version for eg:` to `version; e.g.,` or `version; for example,`
* Delete `, etc.` ("for example" or "e.g." implies a non-exhaustive list; `etc.` is not an example) and insert `.`

Below:

* Change `meta data` to `metadata`
* Again, change `for eg:` to `; e.g.,` or `; for example,`
* Again, delete `, etc.` and insert `.`